otter/constants.py

```diff
@@ -36,6 +37,10 @@ def get_service_configs(config):
             'name': config['cloudLoadBalancers'],
             'region': config['region'],
         },
+        ServiceType.CLOUD_ORCHESTRATION: {
```

Comment:
I actually suggested making it mandatory to reflect the fact that cloud servers is also mandatory here -- in order to facilitate normal otter API operation we will need the orchestration service running just as much as we will need cloud servers.